### PR TITLE
[MID-89] refactor specialist address

### PR DESCRIPTION
**Pull request by IvanKoshan**
PR for this [ticket](https://trello.com/c/YrKd4mPD/89)

### CODE REVIEW

**Review by alex-popov-tech — APPROVED** [review]

File: prisma/seed.mjs [hunk 1]
```diff
@@ -67,8 +56,8 @@ function randomSpecialist({ districts, specializations, therapies }) {
     yearsOfExperience: faker.number.int({ min: 1, max: 30 }),
     // take one of these
     formatOfWork: faker.helpers.arrayElement(['BOTH', 'ONLINE', 'OFFLINE']),
-    placesOfWork: {
-      create: randomPlacesOfWork,
+    addresses: {
+      create: randomAddresses,
```
Comment (watchlar2000) [inline prisma/seed.mjs:60]: Shouldn't "randomAddresses" function be called?

File: src/app/admin/_components/Specialist/Create.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ export function SpecialistCreate() {
         >
```
Comment (watchlar2000) [inline src/app/admin/_components/Specialist/Create.js]: Could you please remove two lines 36 and 37 which are now commented out? I forgot to delete it while working on the specialist creation form.

File: prisma/seed.mjs (hunk 1 above)
Comment (IvanKoshan) [reply prisma/seed.mjs:60]: randomAddresses is an array of addresses that gets generated right above in this function

File: src/app/admin/_components/Specialist/Create.js (hunk 2 above)
Comment (IvanKoshan) [reply src/app/admin/_components/Specialist/Create.js]: done

File: prisma/seed.mjs (hunk 1 above)
Comment (watchlar2000) [reply prisma/seed.mjs:60]: Oh I see. Ok then

**Review by watchlar2000 — APPROVED** [review]

**Review by yataras — APPROVED** [review]

**Pull request merged**